src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -234,9 +234,19 @@ public Task CallFunctionAsync(string functionName, params object[] parameters)
 
             // TODO: Support for versioning
             string version = DefaultVersion;
-            this.config.AssertActivityExists(functionName, version);
+            FunctionType functionType = this.config.GetFunctionType(functionName, version);
 
-            Task<TResult> callTask = this.innerContext.ScheduleTask<TResult>(functionName, version, parameters);
+            Task<TResult> callTask = null;
```

Comment:
>null [](start = 37, length = 4)

Remove the null initializer and add a default case to the switch statement which throws if we don't recognize the function type.